### DISCUSSION TITLE
Anyone from Korea?

### DISCUSSION THREAD

**Discussion by seerge** [seerge/g-helper#369] [General]
Question not related to g-helper. But I was planning to get G14 2023 (next to my G14 2022), and 4080 model seems like a sweet spot.
Unfortunately in NL you can't buy that version at all, and in neighbouring Germany it costs 3500 EUR. 

I have noticed that in Korea G14 4080 is waaay cheaper, like 2100 EUR https://kr.store.asus.com/rog/90nr0dh2-m002r0-ga402xz-n2046.html (despite this model is w/o preinstalled windows or anime matrix, i don't really care)

Are there any hidden taxes (that you need to pay afterwards) ?

**Comment by cheonmux** [comment]
Hi,  @seerge 

Korean here. That is final price.

Our law requires that seller only display the final price with VAT included (10%) . There is no hidden costs.

**Comment by envyska** [comment]
NO 4090 / mini LED / 7735hs models in KR, but we have insane cheapos :) . Those prices are not fraud. Asus might set regional price.. something like 1 eur = 1 pound = 1 dollar = 1000 won ignoring actual exchange rate.

**Reply by seerge** [reply]
@envyska lucky you
